src/vm/database/clarity_db.rs

```diff
@@ -279,15 +289,18 @@ impl BurnStateDB for NullBurnStateDB {
     }
 
     fn get_pox_prepare_length(&self) -> u32 {
-        panic!("NullBurnStateDB should not return PoX info");
+        1
```

Comment:
Why did you remove all of these panics?  If it's for test compatibility, we have a separate `UnitTestBurnStateDB` for this (IIRC use `NullBurnStateDB` in production, or at least used to).

Reply:
Yep -- seconding this: the NullBurnStateDB should continue to panic here. Any test cases that are failing due to this should use the UnitTestBurnStateDB instead.

Reply:
I switched this back. We'll see which tests are failing on GitHub and follow up in another round.

Reply:
@kantai pushed the fixes for this so these tests are working now.